listen_to_mqtt.sh

```diff
@@ -51,13 +52,17 @@ listen_to_mqtt() {
           log_notice "$(cat /share/tesla_ble_mqtt/${vin}_public.pem)"
           log_warning "Private and Public keys were generated; Next:
        1/ Remove any previously deployed BLE keys from vehicle before deploying this one
-       2/ Wake the car up with your Tesla App
-       3/ Push the button 'Deploy Key'"
+       2/ Open the Tesla App on your smartphone and make sure the car is awake.Wake the car up with your Tesla App
```

Comment:
I see duplicated text there: `Open the Tesla App on your smartphone and make sure the car is awake.Wake the car up with your Tesla App`

Reply:
I do too and I didn't drink alcohol, I guess I need to fix that one :)

Reply:
Done.

Reply:
Hmmm still seing. Did you push your commits?